Modify rule S5324: Add Kotlin support (#127)
SonarSource/rspec: rules/S5324/recommended.adoc

== Recommended Secure Coding Practices

* Use internal storage whenever possible as the system prevents other apps from accessing this location.
* Only use external storage if you need to share non-sensitive files with other applications.

> * If your application have to use the external storage to store sentitive data, make sure it encrypts the files using https://developer.android.com/reference/androidx/security/crypto/EncryptedFile[EncryptedFile].

Reviewer: Fix typos:

```suggestion
* If your application has to use the external storage to store sensitive data, make sure it encrypts the files using https://developer.android.com/reference/androidx/security/crypto/EncryptedFile[EncryptedFile].
```

> * Data comming from external storage should always be considered untrusted and should be validated.

Reviewer: Fix typo:

```suggestion
* Data coming from external storage should always be considered untrusted and should be validated.
```

> * As some extrernal storage support can be removed, make sure to never store files that are critical for the usability of your application into it.

Reviewer: Fix typo:

```suggestion
* As some external storage support can be removed, make sure to never store files that are critical for the usability of your application into it.
```
